zhmccli/_cmd_ldap_server_definition.py

```diff
@@ -353,12 +353,12 @@ def cmd_ldapdef_update(cmd_ctx, ldapdef_name, options):
                format(lsd=ldapdef_name))
 
 
-def cmd_ldapdef_delete(cmd_ctx, ldapdef_name):
+def cmd_ldapdef_delete(cmd_ctx, ldapdef):
     # pylint: disable=missing-function-docstring
 
     client = zhmcclient.Client(cmd_ctx.session)
     console = client.consoles.console
-    ldapdef = find_ldapdef(cmd_ctx, console, ldapdef_name)
+    ldapdef = find_ldapdef(cmd_ctx, console, ldapdef)  # Modified argument name
```

Comment:
This reuses the same name for a different type. That causes a pylint error (when the test workflow runs - not sure why it did not run on your PR). I prefer keeping `ldapdef_name`.